Cargo.toml

```diff
@@ -24,3 +24,6 @@ byte = "0.2.4"
 websocket = "0.26.2"
 thiserror = "1.0"
 native-tls = "0.2.7"
+
+[dev-dependencies]
+cargo-tarpaulin = "0.18.0"
```

Comment:
Could you please use version `0.18.0-alpha3` as this is the one I am using inside the GitHub Action. That version has some bugfixes concerning uncovered lines according to [#351](https://github.com/xd009642/tarpaulin/issues/351).